ipfs-cluster-service/main.go

```diff
@@ -210,15 +210,14 @@ configuration.
 					Name:  "custom-secret, s",
 					Usage: "prompt for the cluster secret",
 				},
-				cli.StringFlag{
-					Name:   "env-cluster-secret",
-					EnvVar: "CLUSTER_SECRET",
-					Hidden: true,
-				},
 			},
 			Action: func(c *cli.Context) error {
-				initConfig(c.GlobalBool("force"), !c.Bool("custom-secret"),
-					c.String("env-cluster-secret"))
+				envSecret, envSecretDefined := os.LookupEnv("CLUSTER_SECRET")
+				if envSecretDefined {
```

Comment:
Should we have a function which takes:

```
func userProvidedSecret(customSecret bool) (secret []byte, defined bool) {
var secret string
if customSecret {
  secret =  prompt user
} else if CLUSTER_SECRET is defined
  secret = value
} else {
  return empty, false
}

if secret = "" {
  return []byte{}, true
} else {
  decode
  return decoded secret, true
}
```

And then if defined == true, overwrite the configuration-generated one. Otherwise do nothing. ?